ts/WoltLabSuite/Core/Form/Builder/Field/Controller/FileProcessor.ts

```diff
@@ -8,8 +8,16 @@
 import WoltlabCoreFileElement from "WoltLabSuite/Core/Component/File/woltlab-core-file";
 import { getPhrase } from "WoltLabSuite/Core/Language";
 import { deleteFile } from "WoltLabSuite/Core/Api/Files/DeleteFile";
-import { formatFilesize } from "WoltLabSuite/Core/FileUtil";
 import DomChangeListener from "WoltLabSuite/Core/Dom/Change/Listener";
+import {
+  fileInitializationFailed,
+  insertFileInformation,
+  removeUploadProgress,
+  trackUploadProgress,
+} from "WoltLabSuite/Core/Component/File/File";
+import { clearPreviousErrors } from "WoltLabSuite/Core/Component/File/Upload";
+
+const _data = new Map<string, FileProcessor>();
```

Comment:
```suggestion
type FileId = string;
const fileProcessors = new Map<FieldId, FileProcessor>();
```

This avoid the generic name and makes it immediately clear what it will contain.